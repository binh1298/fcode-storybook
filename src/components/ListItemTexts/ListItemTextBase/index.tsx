import {
    ListItemText as MaterialListItemText,
    ListItemTextProps as MaterialListItemTextProps,
} from "@material-ui/core";

export interface ListItemTextBaseProps extends MaterialListItemTextProps {}

const ListItemTextBase = (props: ListItemTextBaseProps) => {
    return <MaterialListItemText {...props} />;
};

export default ListItemTextBase;
