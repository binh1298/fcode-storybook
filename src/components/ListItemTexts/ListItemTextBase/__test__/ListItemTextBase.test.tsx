import ListItemTextBase, { ListItemTextBaseProps } from "..";

import { render, screen } from "@testing-library/react";

describe("<ListItemTextBase />", () => {
    const props: ListItemTextBaseProps = {
        children: "Test",
    };

    beforeEach(() => {
        render(<ListItemTextBase {...props} />);
    });

    it("should exist children", () => {
        expect(screen.getByText("Test")).toBeInTheDocument();
    });
});
