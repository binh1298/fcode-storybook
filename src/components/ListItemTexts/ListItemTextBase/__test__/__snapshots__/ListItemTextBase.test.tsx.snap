// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonBase /> should match snapshot 1`] = `
<div>
  <div
    class="MuiListItemText-root"
  >
    <span
      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
    >
      <div>
        Test
      </div>
    </span>
  </div>
</div>
`;
